legend-depot-server/src/main/java/org/finos/legend/depot/server/LegendDepotServer.java

```diff
@@ -42,19 +44,24 @@ public static void main(String... args) throws Exception
         new LegendDepotServer().run(args);
     }
 
-
     @Override
     protected GuiceBundle<DepotServerConfiguration> buildGuiceBundle()
     {
-
         return GuiceBundle.defaultBuilder(DepotServerConfiguration.class)
                 .modules(new InfoPageModule())
                 .modules(new DepotServerModule())
                 .modules(new DepotServerResourcesModule())
+                .modules(new PureModelContextModule())
                 .modules(new StoreMongoModule())
                 .modules(new ReadOnlyServicesModule())
                 .modules(new TracingModule())
                 .build();
     }
 
+    @Override
+    public void run(DepotServerConfiguration configuration, Environment environment)
+    {
+        super.run(configuration, environment);
+        environment.jersey().register(LegendDepotServerJacksonProvider.class);
```

Comment:
this will not do wht you intent as it is server wide and it needs to be per type, ei one for PMCD , another for everything  else.